src/seahawk_deck/seahawk_deck/motion_controller.py

```diff
@@ -22,14 +20,17 @@ def __init__(self):
         self.motor_pub = self.create_publisher(Float32MultiArray, 'drive/motors', 10)
         self.subscription = self.create_subscription(Joy, 'joy', self._callback, 10)
     
-    def thrust_control(self, direction1:float, direction2:float)->float:
+    # Calculates what a thruster should output based on multiple input values
+    def combine_input(self, direction1:float, direction2:float)->float:
         """Add two directions in such a way that they do not fall outside [-1, 1]"""
-        if (direction1 >= 0 and direction2 >= 0):
+        if (direction1 >= 0 and direction2 >= 0): # If both input values are positive (0 included)
+            # Combines decimal percentage values based on a probability union operation to determine what the thruster should output
             return direction1 + direction2 - (direction1 * direction2)
-        elif (direction1 < 0 and direction2 < 0):
+        elif (direction1 < 0 and direction2 < 0): # If both input values are negative
+            # Tweaked probability union operation similar to above to work with negative values
             return direction1 + direction2 + (direction1 * direction2)
-        else:
```

Comment:
###### Severity: Minimum ######

I like to leave the trailing `else:` block. The previous and new versions are logically identical, but it suggests to the reader that the else-return is a final possibility in the above conditional tests. -- It indicates they're all one unit.